src/monaca.js

```diff
@@ -759,6 +759,90 @@
+   *     },
+   *     function(err) {
+   *       //Cannot build the project
+   *     }
+   *   );
+   */
+  Monaca.prototype.checkBuildAvailability = function(projectId, platform, buildType) {
+    var deferred = Q.defer();
+
+    if (typeof platform === undefined) {
+      deferred.resolve();
+      return deferred.promise;
+    } else if (typeof buildType === undefined) {
+      deferred.reject(new Error("missing --platform parameter."));
+      return deferred.promise;
+    }
+
+    this._get('/project/' + projectId + '/can_build_app')
+    .then(
+      function(response) {
+        var data;
+
+        try {
+          data = JSON.parse(response);
+        } catch (err) {
+          return deferred.reject(new Error(JSON.stringify(err)));
+        }
+
+        if (data.status === 'ok') {
+          var platformContent = data.result[platform];
+
+          if (!platformContent) {
+            return deferred.reject(new Error("specified platform is not supported."));
+          } else if (platformContent.has_remaining_slot !== true) {
+            return deferred.reject(new Error("you cannot build more apps today."));
+          } else if (platformContent.is_start_file_exist !== true) {
+            return deferred.reject(new Error("start file doesn't exist."));
+          } else if (platformContent.manifest_error) {
+            return deferred.reject(new Error("there is an error with the manifest."));
+          } else if (platformContent.has_assets_encrypt_password) {
+            return deferred.reject(new Error("the project has encrypted assets."));
+          } else if (platformContent.has_inapp_updater_setting) {
+            return deferred.reject(new Error("the project has inApp updater plugin."));
+          } else if (platformContent.can_build_for[buildType] !== true) {
+            return deferred.reject(new Error(platform + " " + buildType + " build is not supported."));
+          } else if (platform === 'android') {
+            if (platformContent.is_versionname_valid !== true) {
+              return deferred.reject(new Error("version name invalid."));
+            } else if (buildType === 'release' && platformContent.has_keysetting !== true) {
+              return deferred.reject(new Error("missing key setting"));
+            }
+          } else if(platform ==='ios') {
+            if (platformContent.info_plist_error) {
+              return deferred.reject(new Error("info_plist error."));
+            } else if (platformContent.has_splash_and_icons !== true) {
+              return deferred.reject(new Error("missing splashscreen and/or icons."));
+            } else if (eval('platformContent.has_' + buildType +'_provisioning') !== true) {
+              return deferred.reject(new Error("missing " + buildType + "_provisioning file."));
+            } else if (eval('platformContent.' + buildType +'_provisioning_error') === true) {
+              return deferred.reject(new Error(buildType + "_provisioning file error."));
```

Comment:
Error in buildType provisioning file. Please upload again from remote build settings.